dist/index.html

```diff
@@ -1,10 +1,30 @@
-<!DOCTYPE html>
-<html><head><script defer src="bundle.js"></script></head>
+<html>
   <head>
-    <meta charset="utf-8">
-    <title>Webpack Template</title>
-    <meta name="description" content="Some description">
-  <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta charset='utf-8' />
+    <title>To-do List</title>
+    <meta
+      name='description'
+      content='Some description'
+    />
+    <meta name='viewport' content='width=device-width, initial-scale=1' />
+    <script
+      src='https://kit.fontawesome.com/219d851abe.js'
+      crossorigin='anonymous'
+    ></script>
+  <script defer src="bundle.js"></script></head>
   <body>
+    <section class='todo-list'>
+      <div class='header-container'>
+        <h1>Today's To Do</h1>
+        <span><i class='fa-solid fa-arrows-rotate'></i></span>
+      </div>
+      <div class='input-container'>
+        <input class='input-text' type='text' placeholder='Add to your list' />
+        <i class='fa-solid fa-arrow-left'></i>
```

Comment:
[ Suggestion Change ]
This change is not a major one to ask for required changes but I trust you'll work on it to help prevent you in your next milestone 👍 

- [ ] The icon you're using here is meant to go back mostly when one clicks on it, it takes the user back to the previous screen, but what you really wanted to use here was the "Enter Key" icon, that tells when the user hit that key, the newly todo text in the input field will get submitted. Kindly search for the enter key icon and change this one.

**Your design**
![image](https://user-images.githubusercontent.com/50941074/192658868-050e00a5-0a8a-4ed9-8062-d728ba6de0ad.png)

**The template design**
![image](https://user-images.githubusercontent.com/50941074/192658934-2ec31f17-ddc9-44ae-8ade-28823a851578.png)